src/storage/ibm/storwize/ssh/mode/eventlog.pm

```diff
@@ -88,7 +89,7 @@ sub run {
         $self->{output}->output_add(
             long_msg => sprintf(
                 '%s : %s - %s', 
-                scalar(localtime($_->{last_timestamp})),
+                scalar(localtime(Time::Piece->strptime($_->{last_timestamp}, '%y%m%d%H%M%S'))),
```

Comment:
Better use `POSIX::strftime('%Y%m%dZ%H%M%S', xxx)` as in apps/pfsense/fauxapi/custom/api.pm